<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterCreator_LeftGuidesWidget</class>
 <widget class="QWidget" name="CharacterCreator_LeftGuidesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>129</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblPage_2">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>41</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;职业信仰&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPage_1">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>21</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;基本资料&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblArrow">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>3</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="coeus_client_qt.qrc">:/coeus_client_qt/right_arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblPage_5">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>101</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;创建完成&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPage_0">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>3</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>进入世界</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPage_3">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>61</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;性格偏好&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPage_4">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>81</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;个人史诗&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="coeus_client_qt.qrc"/>
 </resources>
 <connections/>
</ui>
